### DISCUSSION TITLE
(Linux) Pacstall

### DISCUSSION THREAD

**Discussion by trymeouteh** [ferraridamiano/ConverterNOW#145] [Ideas]
Please add this app into the pacstall repository. Pacstall is the AUR for Ubuntu which will allow Ubuntu users to easily install the app and receive updates very shortly after an update is released.

https://pacstall.dev/

**Comment by ferraridamiano** [comment]
At the moment you can install this app on linux in many ways (flatpak, snap, building from source) and according to me adding more ways of installing the app is not a priority. It is a FOSS project, if you want to add this app to Pacstall you can do it 😉
